macros/datetime/date_spine.sql

```diff
@@ -17,8 +17,21 @@
 
 {%- endmacro %}
 
+{# private, don't call from anywhere else #}
+{%- macro date_spine__parse_end_date(end_date) -%}
+  {{ adapter_macro('dbt_utils.prep_end_date_', end_date) }}
```

Comment:
I think this should be
```suggestion
  {{ adapter_macro('dbt_utils.date_spine__parse_end_date', end_date) }}
```